### PR TITLE
docs: update the public roadmap

### PR DESCRIPTION
**Pull request by mgechev**
Change status of existing projects and add new projects we're working
on.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Review by jelbourn — APPROVED** [review]
One thing missing is @mmalerba's work on new cdk primitives- I'll let him weight in on what we'd want to say here

Another thing potentially work mentioning is our ongoing a11y work on components. In particular, we plan on improving things related to [non-text contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html) this year.

otherwise LGTM

File: aio/content/guide/roadmap.md [hunk 1]
```diff
@@ -29,15 +34,11 @@ We're evaluating components in Angular Material against accessibility standards
 
 ### Remove legacy [View Engine](guide/glossary#ve)
 
-After the transition of all our internal tooling to Ivy is completed, we will remove the legacy View Engine for reduced Angular conceptual overhead, smaller package size, lower maintenance cost, and lower codebase complexity.
-
-### Launch advanced compiler diagnostics
-
-Extend the diagnostics of the Angular compiler outside type checking. Introduce other correctness and conformance checks to further guarantee correctness and best practices.
+After the transition of all our internal tooling to Ivy is completed, we will remove the legacy View Engine for reduced Angular conceptual overhead, smaller package size, lower maintenance cost, and lower codebase complexity.
```
Comment (AndrewKushnir) [inline aio/content/guide/roadmap.md:37]: I think we can move this item to the "done" section :)

**Comment by mmalerba** [comment]
We can add a section for the CDK components I'm working on. I would say something like:

> We're working on new CDK primatives to facilitate creating custom components based on the WAI-ARIA design patterns for [Menu & Menubar](https://www.w3.org/TR/wai-aria-practices-1.1/#menu), [Listbox](https://www.w3.org/TR/wai-aria-practices-1.1/#Listbox), [Combobox](https://www.w3.org/TR/wai-aria-practices-1.1/#combobox), and [Dialog](https://www.w3.org/TR/wai-aria-practices-1.1/#dialog_modal)

We plan to have menu & menubar available by 14.0.0 final, and hopefully the others as well. I have a long vacation coming up, but talked to @andrewseguin and I think the plan is to have someone take over where I leave off in getting them ready for 14.

File: aio/content/guide/roadmap.md [hunk 2]
```diff
@@ -1,23 +1,28 @@
...
 
 ## In progress
 
-### Better developer ergonomics with strict typing for `@angular/forms`
+### Rollout strict typings for `@angular/forms`
 
-We will work on finding a way to implement stricter type checking for reactive forms with minimal backward incompatible implications. This way, we let developers catch more issues during development time, enable better text editor and IDE support, and improve the type checking for reactive forms.
-### Simplified Angular mental model with optional NgModules
+In Q4 2022 we designed a solution for introducing strict typings for forms and in Q1 2022 we concluded the corresponding [request for comments](https://github.com/angular/angular/discussions/44513). Currently, we're implementing a rollout strategy with an automated migration step that will enable the improvements for existing projects. We're first testing the solution with 2,500+ projects at Google to ensure a smooth migration path for the external community.
```
Comment (eneajaho) [inline aio/content/guide/roadmap.md:13]:
```suggestion
In Q4 2021 we designed a solution for introducing strict typings for forms and in Q1 2022 we concluded the corresponding [request for comments](https://github.com/angular/angular/discussions/44513). Currently, we're implementing a rollout strategy with an automated migration step that will enable the improvements for existing projects. We're first testing the solution with 2,500+ projects at Google to ensure a smooth migration path for the external community.
```

Shouldn't it be 2021?

**Review by madleinas — APPROVED** [review]
Agree that we should also include the CDK work that Miles is working on. 

Otherwise LGTM

**Comment by mgechev** [comment]
The CDK work is there too. Adding merge label.

**Comment by dylhunn** [comment]
This PR was merged into the repository by commit 0f9b3c67b4e4af5ad0d1003421ae95dba7b1980e.

**Pull request closed without merging**